docs/reference/encrypted-int4-domain.md

```
@@ -0,0 +1,156 @@
+ORDER BY caveat: sort by the OPE-key extractor:
+
+```sql
+ORDER BY eql_v2.eql_v2_int4_ord_ope_ope_key(col::jsonb)
+```
+
+## Inlining chain
+
+For `_ord_ope` range queries, the planner sees a clean three-layer
+inline chain through to a built-in `bytea` compare:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)